tidb-cloud/vector-search-functions-and-operators.md

```diff
@@ -222,7 +222,7 @@ Examples:
 VEC_L2_NORM(vector)
 ```
 
-Calculates the [L2 norm](https://en.wikipedia.org/wiki/Norm_(mathematics)) (Euclidean norm) of a vector using the following formula:
+Calculates the [L2 norm](<https://en.wikipedia.org/wiki/Norm_(mathematics)>) (Euclidean norm) of a vector using the following formula:
```

Comment:
Why there is a pair of '<>' at the margin of the link?

Reply:
It is added by the formatter by default. Although the rendering is fine without this `<>`, I believe the formatter is right for good: https://stackoverflow.com/questions/13824669/how-do-you-write-a-link-containing-a-closing-bracket-in-markdown-syntax